docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,210 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you may use the
+{{gui}} or the OpenStack CLI. Following both approaches, let us
+demonstrate the creation of a new server with vGPU support.
+
+## Prerequisites
+
+Although not necessary, it certainly helps a lot to have some
+familiarity with [creating servers in
+{{brand}}](/howto/openstack/nova/new-server). Additionally, if you
+prefer to work with the OpenStack CLI, then make sure to properly
+[enable it first](/howto/getting-started/enable-openstack-cli).
+
+## Instantiating a vGPU server
+
+=== "{{gui}}"
+    Fire up your favorite web browser, navigate to the
+    [{{gui}}](https://{{gui_domain}}) start page, and log into your
+    {{brand}} account.
+
+    Create your new vGPU server, working almost exactly as you would when
+    creating a "normal" server. There are two steps that you should pay
+    special attention to, though.
+        
+    First, the image for the new server should be the `Ubuntu 22.04 NVGRID
+    x86_64`; that is your only option for a server with vGPU support.
+        
+    ![Select image with vGPU support](assets/new-vgpu-server/shot-01.png)
```

Comment:
The screenshot isn't included in the PR (this is more of a note-to-self, as I presume that this is why the PR is marked as a draft).

Reply:
Yes, I expect to have two screenshots exactly, and I prefer to create those as soon as the `g`-flavors are visible in CCMP.